src/styles/nsw-point-address.scss

```diff
@@ -0,0 +1,4 @@
+.ob-point-address__container {
+  container-name: ob-point-address__container;
+  container-type: inline-size;
+}
```

Comment:
Liquor licence already have a container somewhere?

Reply:
Its container name is "liquor license" specific. Can't change it without potentially effecting peoples custom CSS.